### ISSUE TITLE
All releases prior to 1.4.18 are tagged to 346c3a6

### ISSUE THREAD

**Issue by echometerain** [Crazy-Crew/CrazyEnvoys#76]
Here are the commits they should point to:

1.4.17: 366553d
1.4.16: af816c3
1.3.16: 4b98383
1.3.15: 91ef4c6
1.3.14: 6d933b2
1.3.13: c5fc6aa
1.3.12: c6c1c82
1.3.11: af22f30
1.3.10: 17f7220
1.3.9: f18d1d0
1.3.8: d9a5b94
1.3.7: 91371d8
1.3.6: 36dc127

**Comment by RyderBelserion** [comment]
What?

**Comment by echometerain** [comment]
take v1.4.17 as an example
![image](https://user-images.githubusercontent.com/70437021/202801910-4d8fbdb3-f8e5-42f2-9e3d-ca93d94d6585.png)
the release page and the source code download points to the commit 346c3a6 while the actual `version bump` commit happened 2 months ago in 366553d
github uses the commit hash on releases tags to decide which source code `.zip` you download. When I tried to download the source for v1.3.9, github actually downloaded the version of v1.4.18 in 346c3a6

**Comment by RyderBelserion** [comment]
The version bump 2 months ago was not released so I release v1.4.17 after & set the version bump tag, The source code is outdated and doesn't line up with the jar added since that would've been based on 346c3a6.

**Comment by RyderBelserion** [comment]
> The version bump 2 months ago was not released so I release v1.4.17 after & set the source tag to [366553d](https://github.com/Crazy-Crew/CrazyEnvoys/commit/366553d4614ca58f7ee4f475b080df9c1830343b), The source code is outdated and doesn't line up with the jar added since that would've been based on [346c3a6](https://github.com/Crazy-Crew/CrazyEnvoys/commit/346c3a6da1a5aecaa6adcd3231c827aa2bd447e2).
> 
> Vice versa for other related version bumps.

What else github does beyond that though is far out of my control as we can't edit commit tags on current releases so if an older release gets update. It takes the latest commit...

**Comment by echometerain** [comment]
reassigning the releases should just be:
```
git checkout f18d1d0
git tag -f -a 1.3.9
... (repeat for other tags)
git push -f --tags
```
https://stackoverflow.com/questions/8044583/how-can-i-move-a-tag-on-a-git-branch-to-a-different-commit
But since the actual release commits are not the version bump commits I guess it won't be worth your time to try to find all of them.
I don't play minecraft actually, a friend just asked me to compile `1.3.9` (which didn't work lmao the maven repos are down)  and I had to scour through a sea of commits. I got bored today so I decided to find all of them.
